docs/source/user_guide/types.rst

```diff
@@ -215,3 +215,14 @@ You can also check the underlying PySpark data type of `Series` or schema of `Da
      |-- d: double (nullable = false)
      |-- s: string (nullable = false)
      |-- b: boolean (nullable = false)
+
+.. note::
+
+    Koalas currently does not support multiple types of data in single column.
+
+.. code-block:: python
```

Comment:
Can you add this example inside the note?

Reply:
Addressed. Thanks!